src/watcher/garbageCollector.ts

```diff
@@ -84,9 +83,9 @@ export class AppointmentStoreGarbageCollector extends StartStopService {
                 // an error here means that we were likely unable to collect all, or some of, the appointments
                 // consecutive errors could mean we have a systematic problem, or connection issues with the store
                 // in either case the problem is very serious - so we stop the GC
-                logger.error("GC: Unexpected error.");
-                logger.error(`GC: Consecutive errors: ${this.consecutiveErrors}.`);
-                logger.error(doh);
+                this.logger.error("GC: Unexpected error.");
```

Comment:
Remove the "GC" string from the logs in this class

Reply:
Removed.